F/Fmt/build_tarballs.jl

```diff
@@ -0,0 +1,53 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Fmt"
+version = v"9.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/fmtlib/fmt.git", "a33701196adfad74917046096bf5a2aa0ab0bb50")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+
+cd $WORKSPACE/srcdir/fmt
+mkdir build
+cd build
+
+cmake \
+-DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN%.*}_gcc.cmake \
```

Comment:
Why using GCC everywhere?

Reply:
I run into clang errors on mac without this. Linux and windows build fine either way but Mac is the problem child in this case.

Reply:
Can you elaborate on _what_ errors you got?

Reply:
ld: library not found for -lc++fs
clang-13: error: linker command failed with exit code 1 (use -v to see invocation)

Above is the error encountered on mac (both x86_64 and aarch64).